packages/usa-input-mask/src/usa-input-mask.twig

```diff
@@ -0,0 +1,42 @@
+<h3 class="site-preview-heading">Social Security Number Input Mask</h3>
+<label class="usa-label" for="ssn">Social Security Number</label>
+<div class="usa-hint flex" id="ssnHint">For example: XXX XX XXXX</div>
+<input 
+    id="ssn" 
+    type="text" 
+    inputmode="numeric" 
+    name="ssn" 
+    placeholder="XXX XX XXXX" 
+    pattern="^(?!(000|666|9))\d{3} (?!00)\d{2} (?!0000)\d{4}$" 
+    class="usa-input usa-masked" 
+    title="9-digit social security number" 
+    aria-describedby="ssnHint" 
+>
+
+<h3 class="site-preview-heading">US Telephone Number Input Mask</h3>
+<label class="usa-label" for="tel">US Telephone Number</label>
+<div class="usa-hint" id="pnHint">For example: (XXX) XXX-XXXX</div>
```

Comment:
The screen reader readout isn't very helpful here. On VoiceOver, users hear "For example XXXXXXXXX". The placeholder in this component does show the visual example in the hint already - could we instead rewrite this hint to give clear instructions for what to do? I am thinking of something like "Provide your 10-digit telephone number, starting with the area code."

If we do not want to give that hint to visual users, could we place instructional copy in an aria-label on the `<input>` instead of using `aria-describedby`?